Paintroid/src/main/res/layout/color_chooser_layout_rgbview.xml

```diff
@@ -184,7 +184,34 @@
                 android:textAppearance="?android:attr/textAppearanceSmall"
                 android:textColor="@color/pocketpaint_colorAccent"
                 tools:ignore="HardcodedText" />
+        </TableRow>
+
+        <TableRow
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:paddingBottom="12dp"
+            android:paddingTop="12dp">
+
+            <TextView
+                android:id="@+id/color_chooser_color_rgb_textview_hex"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_gravity="center_vertical"
+                android:gravity="center_vertical"
+                android:text="@string/color_hex"
+                android:textColor="@color/pocketpaint_color_chooser_black"
+                android:paddingStart="0dp"
+                android:paddingEnd="5dp" />
 
+            <EditText
+                android:id="@+id/color_chooser_color_rgb_hex"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_gravity="center_vertical"
+                android:hint="#"
+                android:inputType="textNoSuggestions"
+                tools:ignore="HardcodedText"
+                android:focusable="false"/>
```

Comment:
Are you sure about this?
This does not allow me to input any value: https://streamable.com/p4bxv

Reply:
Correct. That didn't go as planned.
I'm looking for a different way to stop the autoscroll to the EditText.